### PR TITLE
(fix) Fix conditional access check affecting patientIdentifier type fields 

### PR DESCRIPTION
**Pull request by gitcliff**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
This PR is addressing the issue whereby after saving a form of patient identifier type, upon subsequent attempts to reload the form in the form-renderer the schema fails to load.
## Screenshots
https://www.loom.com/share/7d5e92a9e98c423cb05270701f7539cb

### CODE REVIEW

**Comment by samuelmale** [comment]
The title of the PR mentions adding some kind of validation but I don't see any validation specific logic, all I see is conditional chaining. @gitcliff is this a WIP?

**Comment by gitcliff** [comment]
@samuelmale this PR covers the initial part of failure to load the form schema for patient identifier questions, this was discovered during the patientIdentifier validation for testing. Going to raise another PR that cover the patient Identifier validation.
Thanks

**Comment by samuelmale** [comment]
@gitcliff the PR title doesn't clearly define what the changes. Is fixing a reference to null/undefined prop or it's enabling  loading of the form schema for patient Identifier type Questions? The title is a bit ambiguous.

**Comment by gitcliff** [comment]
> @gitcliff the PR title doesn't clearly describe what the changes are about. Is it fixing a reference to null/undefined prop or it's enabling loading of the form schema for patient Identifier type Questions? The title is a bit ambiguous.

@samuelmale its **Fixing reference to null prop when loading the form schema for patient Identifier type Questions **

**Review by hadijahkyampeire — APPROVED** [review]
LGTM

File: src/utils/patient-identifier-helper.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ export const saveIdentifier = (patient: fhir.Patient, patientIdentifier: Patient
 };
 
 export const getPatientLatestIdentifier = (patient: fhir.Patient, identifierType: string) => {
-  const patientIdentifiers = patient.identifier;
-  return patientIdentifiers.find((identifier) => {
+  const patientIdentifiers = patient?.identifier;
+  return patientIdentifiers?.find((identifier) => {
```
Comment (samuelmale) [inline src/utils/patient-identifier-helper.ts:15]: @gitcliff do we need the `patientIdentifiers` state?

```suggestion
  return patient?.identifier?.find((identifier) => {
```

**Review by samuelmale — APPROVED** [review]
Just one minor suggestion, but this is good to go.

File: src/utils/patient-identifier-helper.ts (hunk 1 above)
Comment (samuelmale) [reply src/utils/patient-identifier-helper.ts:15]: Plus is it possible for the patient to null/undefined?

**Review by denniskigen — APPROVED** [review]

**Pull request merged**